apps/studio/package.json

```diff
@@ -28,7 +28,7 @@
     "test-dev:unit": "dotenv -e .env.test vitest",
     "test-start": "start-server-and-test start http://127.0.0.1:3000 test",
     "postinstall": "npm run generate && npm run build:theme",
-    "storybook": "storybook dev -p 6006",
+    "storybook": "storybook dev -p 6007",
```

Comment:
Did this conflict with another port?

Reply:
yes we have 2 storybooks now - 1 for components, 1 for studio